src/SFA.DAS.EmploymentCheck.Data/Models/LearnerNiNumberValidator.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Net;
+
+namespace SFA.DAS.EmploymentCheck.Data.Models
+{
+    public class LearnerNiNumberValidator : ILearnerNiNumberValidator
+    {
+        public (bool IsValid, string ErrorType) IsValidNino(EmploymentCheckData employmentCheckData)
+        {
```

Comment:
i might not be quite following here, but what's the point of returning bool as well as ErrorType? isn't the fact that `ErrorType` is not null means it's `invalid` i.e. is rhetorical and can just be an auto-property i.e. `IsValid => ErrorType != null` ?

Reply:
just trying to remove a possibility of setting `IsValid` to true without setting the `ErrorType` this way

Reply:
Refactored to just return a string.